pom.xml

```diff
@@ -650,6 +650,8 @@
 
         <scala.base>2.12</scala.base>
         <scala.version>${scala.base}.8</scala.version>
+
+        <doclint>none</doclint>
```

Comment:
usually with have this:
```
        <profile>
            <id>doclint-java8-disable</id>
            <activation>
                <jdk>[1.8,)</jdk>
            </activation>
            <build>
                <plugins>
                    <plugin>
                        <groupId>org.apache.maven.plugins</groupId>
                        <artifactId>maven-javadoc-plugin</artifactId>
                        <configuration>
                            <additionalparam>-Xdoclint:none</additionalparam>
                        </configuration>
                    </plugin>
                </plugins>
            </build>
        </profile>
```
any reason to change? Or then not to remove old doclint way?

Reply:
Well we enforce the build to be java 11, thus why using some 'if(java 8) then xxx' ?

It adds complexity, and duplicated javadoc configuration (one in the profile for java 8 and more, one for non java 8 in the plugin config).

Reply:
Fine, then you can resolve the last part of my comment: remove the old doclint way.
```
mpt/app/pom.xml
mpt/core/pom.xml
mpt/mavenplugin/pom.xml
```